### DISCUSSION TITLE
What can I do to investigate the cause?

### DISCUSSION THREAD

**Discussion by fxzxmicah** [p11-glue/p11-kit#658] [Q&A] [answered]
```
9月 17 22:11:21 HP-Pavilion-PC systemd[1765]: Starting flatpak-session-helper.service - flatpak session helper...
9月 17 22:11:21 HP-Pavilion-PC flatpak-session[3058]: Unable to start p11-kit server: 子进程已退出，代码 2
9月 17 22:11:21 HP-Pavilion-PC systemd[1765]: Started flatpak-session-helper.service - flatpak session helper.

```

**Comment by ZoltanFridrich** [comment, marked as answer]
Hello,

I am not sure how the p11-kit server is called here, but p11-kit outputs error messages to stderr by default (unless suppressed with -q option). I personally would try to check how the p11-kit server is called and try to enable the debug output with --verbose.

**Reply by fxzxmicah** [reply]
https://github.com/flatpak/flatpak/blob/1561e0f39cbc1a157c821f9dd4729f08e9a08550/session-helper/flatpak-session-helper.c#L700
I found this.

**Reply by fxzxmicah** [reply]
Thank you for your inspiration. I have solved it and need to install this package: `p11-kit-server`.
